src/main.js

```diff
@@ -61,9 +61,14 @@ new Vue({
   apolloProvider,
   vuetify,
   beforeCreate() {
-    this.$intercom.boot({
-      user_id: uuidv4()
-    });
+    const userId = this.$route.query.intercomid
+      ? this.$route.query.intercomid
+      : uuidv4();
+    if (userId) {
```

Comment:
won't need to check this since if intercomid is not there, it will be filled with the uuidv4() function